.github/workflows/release.yml

```diff
@@ -16,7 +16,6 @@ jobs:
         ref: master
     - uses: actions/setup-node@v3
       with:
-        node-version: 14
```

Comment:
Same here. The release action has to work, no matter what. So a node dedicated node version has to be set.
However, i am fine if you upgrade to node 16 here